easybuild/easyblocks/l/lapack.py

```diff
@@ -143,6 +143,10 @@ def configure_step(self):
             self.log.info('No BLAS library provided, so only building LAPACK library (no testing).')
             self.cfg.update('buildopts', 'lib')
 
+
+        if os.path.exists(os.path.join(self.cfg['start_dir'], 'lapacke')):
```

Comment:
how about just checking the version? this should be there in LAPACK 3.4.0 and up